src/main/kotlin/Figure.kt

```diff
@@ -0,0 +1,6 @@
+class Figure(val color : Int) {
+  
+    constructor(col : String) : this(if (col[0] == 'w') 1 else -1)
```

Comment:
This "from string" constructor,if needed, should be a Color constructor, I believe.

Reply:
Fixed